1.hello-world/app.js

```diff
@@ -17,9 +17,16 @@ const port = 3000;
 app.get('/order', (_req, res) => {
     fetch(`${stateUrl}/order`)
         .then((response) => {
-            return response.json();
+            if (!response.ok) {
+                throw "Could not get state.";
+            }
+
+            return response.text();
         }).then((orders) => {
             res.send(orders);
+        }).catch((error) => {
+            console.log(error);
+            res.status(500).send();
```

Comment:
it would be good to return error in response body. e.g. res.status(500).send({message: JSON.stringify(error)});

Reply:
Will do.

Reply:
Done. stringify() does not seem to be applicable here since I did not encounter an exception that is a complex type. If I use stringify() we end up with a string inside a string, which look odd.